e_os.h

```diff
@@ -359,10 +359,10 @@ inline int nssgetpid();
 
 # ifndef OPENSSL_NO_SECURE_MEMORY
    /* unistd.h defines _POSIX_VERSION */
-#  if defined(OPENSSL_SYS_UNIX) \
+#  if (defined(OPENSSL_SYS_UNIX) \
       && ( (defined(_POSIX_VERSION) && _POSIX_VERSION >= 200112L)      \
            || defined(__sun) || defined(__hpux) || defined(__sgi)      \
-           || defined(__osf__) )
+           || defined(__osf__) )) || defined(_WIN32)
```

Comment:
Break the `|| defined(_WIN32)` onto the next line.  The indentation is also off with the addition of the initial `(`.

Reply:
I'm hoping I got this right now, can you double check me please?